### ISSUE TITLE
Crash when holding yubikey while in ViewKey

### ISSUE THREAD

**Issue by dschuermann** [open-keychain/open-keychain#1392]
Already in ViewKeyActivity after holding a YubiKey against OK.
If I hold it again -> crash

```
 11700         AndroidRuntime  E  java.lang.AssertionError: should not happen: NfcOperationActivity.onCreate is called instead of onNewIntent!
 11700         AndroidRuntime  E  at org.sufficientlysecure.keychain.ui.base.BaseNfcActivity.onCreate(BaseNfcActivity.java:84)
 11700         AndroidRuntime  E  at org.sufficientlysecure.keychain.ui.ViewKeyActivity.onCreate(ViewKeyActivity.java:146)
 11700         AndroidRuntime  E  at android.app.Activity.performCreate(Activity.java:5990)
 11700         AndroidRuntime  E  at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1106)
 11700         AndroidRuntime  E  at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2309)
 11700         AndroidRuntime  E  at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2418)
 11700         AndroidRuntime  E  at android.app.ActivityThread.access$900(ActivityThread.java:154)
 11700         AndroidRuntime  E  at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1321)
 11700         AndroidRuntime  E  at android.os.Handler.dispatchMessage(Handler.java:102)
 11700         AndroidRuntime  E  at android.os.Looper.loop(Looper.java:135)
 11700         AndroidRuntime  E  at android.app.ActivityThread.main(ActivityThread.java:5293)
 11700         AndroidRuntime  E  at java.lang.reflect.Method.invoke(Native Method)
 11700         AndroidRuntime  E  at java.lang.reflect.Method.invoke(Method.java:372)
 11700         AndroidRuntime  E  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:904)
 11700         AndroidRuntime  E  at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:699)
```

**Comment by Valodim** [comment]
cannot reproduce. BaseNfcActivity:84 also looks like an older version?

**Comment by Valodim** [comment]
probably fixed with #2157, since the special handling in ViewKeyActivity that caused this has been removed.
